scripts/autopush.sh

```diff
@@ -33,10 +33,12 @@ git checkout master
 # Deploy autopush instance
 gsutil cp gs://datcom-control/latest_base_bigquery_version.txt deploy/storage/bigquery.version
 # Import Group
-> deploy/storage/bigtable_import_groups.version
+yq eval -i 'del(.tables)' deploy/storage/base_bigtable_info.yaml
+yq eval -i '.tables = []' deploy/storage/base_bigtable_info.yaml
```

Comment:
More references:
https://github.com/datacommonsorg/website/search?q=bigtable_import_groups
https://github.com/datacommonsorg/mixer/search?q=bigtable_import_groups

Release docs: https://source.corp.google.com/search?q=bigtable_import_groups%20f:datacommons

`<rant> github search is so terrible </rant>`

Reply:
The reference in mixer is not in use anymore (going to delete those).
The references in website is still under development, Alex is working on a new version from that. So leaving that untouched for now.

I will update the release doc.